index.html

```diff
@@ -3244,6 +3294,108 @@ <h3 id="applying">
           </li>
         </ol>
       </section>
+      <section>
+        <h3>
+          <code>min_version</code> member
+        </h3>
+        <p>
+          The <dfn data-lt="application-min-version">min_version</dfn> member
+          of an application object represents the minimum version required for
```

Comment:
_"minimum version required"_

"required" is a bit weird here since referring to an application in the manifest does not "require" anything. Perhaps, "the minimum version of the application that is considered related to this web app."